rest.go

```diff
@@ -13,7 +13,7 @@ import (
 type JSON map[string]interface{}
 
 // RenderJSON sends data as json
-func RenderJSON(w http.ResponseWriter, data interface{}) {
+func RenderJSON(w http.ResponseWriter, data interface{}, code ...int) {
```

Comment:
addind an optional parameter like this feels unusual and confusing. Someday someone will pass multiple codes because the API allows it. The trick allows to keep compatibily but stll a trick.